### PR TITLE
Fix layers displaying in cable stacks

### LINKED ISSUE #4004: Cable coil stack visualizer is a little off
## Description
at around 22/23 cable (still the majority of the cable left) the sprite is just a single strand of cable which doesn't match how much is left

**Screenshots**
image

**Additional context**

### PR DESCRIPTION
**Pull request by Ygg01**
Fixed stackVisualizer to use another rounding method

Fixes #4004 

**Screenshots**
@mirrorcult Is this expected behavior?

https://user-images.githubusercontent.com/1146204/118867342-433b9f80-b8e3-11eb-9afd-418f6314cc10.mp4

**Changelog**

:cl: Ygg01
- fix: StackVisualizer now correctly displays levels

### CODE REVIEW

**Review by mirrorcult — APPROVED** [review]

**Review by PaulRitter — APPROVED** [review]
while i remember the original behaviour in ss13 being this:
one wire -> lvl 1
two wires -> lvl 2
other -> lvl 3

this should be fine too

**Comment by Ygg01** [comment]
> while i remember the original behaviour in ss13 being this:
> one wire -> lvl 1
> two wires -> lvl 2
> other -> lvl 3
> 
> this should be fine too

I can add this behavior as a special case at a later point. I assume most want equal level stacks.

File: Content.Tests/Shared/Utility/ContentHelpers_Test.cs [hunk 1]
```diff
@@ -82,6 +82,58 @@ public class ContentHelpers_Test
...
+                (7, 7, 3, 2),
+                
+                // Testing edge cases
+                (0.1, 6, 5, 0),
+                (-32, 6, 5, 0),
+                (2.4, 6, 5, 1),
+                (2.5, 6, 5, 2),
+                (320, 6, 5, 4),
+            };
+
```
Comment (ShadowCommander) [inline Content.Tests/Shared/Utility/ContentHelpers_Test.cs:136]: This can probably be replaced with [TestCase(-10, 5, 2, 0)] placed above TestEqual in place of [Test]

**Review by ShadowCommander — CHANGES_REQUESTED** [review]

File: Content.Shared/Utility/ContentHelpers.cs [hunk 2]
```diff
@@ -98,5 +98,42 @@ public static int RoundToNearestLevels(double actual, double max, int levels)
...
+        /// <param name="actual">How many out of max elements are there</param>
+        /// <param name="max"></param>
+        /// <param name="levels"></param>
+        /// <returns>The </returns>
+        /// <exception cref="ArgumentException">if level is one or less</exception>
+        public static int RoundToEqualLevels(double actual, double max, int levels)
+        {
+            if (levels <= 1)
+            {
+                throw new ArgumentException("Levels must be greater than 1.", nameof(levels));
```
Comment (ShadowCommander) [inline Content.Shared/Utility/ContentHelpers.cs:123]: Will this throw if the cable RSI only has one layer?

Comment (Ygg01) [reply Content.Shared/Utility/ContentHelpers.cs:123]: Yes. A stack of icons with one layer, is just an icon.

Comment (ShadowCommander) [reply Content.Shared/Utility/ContentHelpers.cs:123]: Something for a different PR then, but it should probably be made to still work even if someone replaced the RSI with only a single layer.

**Review by ShadowCommander — APPROVED** [review]
